docs/notes/bugfix-19377.md

```diff
@@ -0,0 +1 @@
+# Changed an example in the version entry to not operate on alphabetical comparison.
```

Comment:
I would suggest something more generic, e.g. `Updated example in `version` dictionary entry`